.changeset/new-mangos-learn.md

```diff
@@ -0,0 +1,6 @@
+---
+"@10up/eslint-plugin": major
```

Comment:
We aren't actually using this today so this won't trigger a major release on 10up-toolkit.